docs/conf.py

```diff
@@ -203,9 +203,9 @@
     sphinx_gallery_conf = {
         'backreferences_dir': 'generated{}modules'.format(os.sep), # path to store the module using example template
         'filename_pattern': '^((?!skip_).)*$', # execute all examples except those that start with "skip_"
-        'examples_dirs': '..{0}..{0}examples'.format(os.sep), # path to the examples scripts
+        'examples_dirs': '..{}examples'.format(os.sep), # path to the examples scripts
         'gallery_dirs': 'generated{}gallery'.format(os.sep), # path to save gallery generated examples
-        'default_thumb_file': '..{}logo{}sunpy_icon_128x128.png'.format(os.sep, os.sep),
+        'default_thumb_file': '.{}logo{}sunpy_icon_128x128.png'.format(os.sep, os.sep),
```

Comment:
You don need two `os.sep` in format. Though I don't really like to use `os.sep` in this way... I rather use `os.path.join`... but I imagine is the same...